demos/serverless/src/ChimeMeetings.js

```diff
@@ -0,0 +1,57 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+const AWS = require('aws-sdk');
+
+module.exports = class ChimeMeetings {
```

Comment:
Isn;t this the same one as the local server? Can we just have one?

Reply:
I tried to move the ChimeMeetings file to demo level but it is not working to have a single file we will need to add webpack inside serverless demo which i think is out of scope for this ticket.